modules/helper-functions.js

```diff
@@ -209,6 +210,14 @@ const filteredReplyComments = (filter, arr) => {
   return reply;
 };
 
+const tryCatchWrapper = async handler => {
```

Comment:
Explain, which means the name `tryCatchWrapper` if you handle some handler with potential error?